backend/server/routers/planner.py

```diff
@@ -11,33 +11,33 @@
 from fastapi import APIRouter, HTTPException
 from server.config import DUMMY_TOKEN
 from server.routers.courses import get_course
-from server.routers.model import (CourseCode, PlannedToTerm, PlannerData, ProgramTime, UnPlannedToTerm,
-                                  ValidCoursesState, ValidPlannerData)
+from server.routers.model import (CourseCode, Mark, PlannedToTerm, PlannerData, ProgramTime, Storage, UnPlannedToTerm,
+                                  ValidCoursesState, ValidPlannerData, markMap)
 from server.routers.user import get_user, set_user
 from server.routers.utility import get_course_object
 
 MIN_COMPLETED_COURSE_UOC = 6
 
 
-def fix_planner_data(plannerData: PlannerData) -> ValidPlannerData:
+def convert_to_planner_data(user: Storage) -> ValidPlannerData:
```

Comment:
this is to allow me to use the validateTermPlanner code as it was before. It reads the user and converts it to the correct format